Scripts/UninstallGVFS.bat

```diff
@@ -9,11 +9,13 @@ if not exist "c:\Program Files\GVFS" goto :end
 
 call %VFS_SCRIPTSDIR%\StopAllServices.bat
 
+if not "%2"=="--remove-prjflt" goto :uninstallGVFS

```

Comment:
So we'll update our pipeline to call the uninstall script with this parameter on server2016, but leave the inbox version alone on 19h1?

Reply:
Yep and it will actually be the `ReinstallGVFS.bat` that will need to be passed the argument and it calls `UninstallGVFS.bat` passing the argument it got because it also needs to check for the `--remove-prjflt`

Reply:
https://github.com/microsoft/VFSForGit/blob/825202e19568d192ed406083e04be61664a221b8/Scripts/ReinstallGVFS.bat#L24-L29

Reply:
Cool, thanks!